components/canvas/Drawing.module.css

```diff
@@ -1,13 +1,12 @@
 .patternContainer {
   display: grid;
   grid-template-columns: repeat(2, 1fr);
-  grid-template-rows: 0.5fr 0.5fr 2fr 1fr;
+  grid-template-rows: repeat(3, 1fr);
   grid-column-gap: 1rem;
-  grid-row-gap: 2rem;
 }
 
-.patternimg {
-  grid-area: 1 / 1 / 3 / 2;
+.patternimg1 {
```

Comment:
What is the difference between 1 and 2?
It is not usual to work with numbers here